workflows/subwf-rna-single-organism.wdl

```diff
@@ -170,5 +176,12 @@ workflow wf_rna {
         File? share_rna_seurat_umap_mito_plot = seurat.seurat_umap_mito_plot
         File? share_rna_seurat_obj = seurat.seurat_obj
         File? share_rna_plots_zip = seurat.plots_zip
+
+        Int rna_total_reads = log_rna.rna_total_reads
+        Int rna_aligned_uniquely = log_rna.rna_aligned_uniquely
+	Int rna_aligned_multimap = log_rna.rna_aligned_multimap
+        Int rna_unaligned = log_rna.rna_unaligned
+        Int rna_feature_reads = log_rna.rna_feature_reads
+        Int rna_duplicate_reads = log_rna.rna_duplicate_reads
```

Comment:
The naming is not consistent with the other outputs
rna_total_reads
instead of:
share_rna_total_reads